### PR TITLE
Move tabulate and six to separate folders:

### PR DESCRIPTION
**Pull request by ankurankan**
Just to make things more cleaner and adds license for six.

@khalibartan @yashu-seth I have been trying to move six and tabulate to separate dirs. But it's behaving funny or I am doing something silly. Can you guys please have a look and see what's wrong ?
I don't think we even need `__init__` in the subdirs.

### CODE REVIEW

File: pgmpy/extern/__init__.py [hunk 1]
```diff
@@ -1,3 +1,5 @@
-from .tabulate import tabulate
+from .six import six
+from .tabulate.tabulate import tabulate
 
-__all__ = ['tabulate']
+__all__ = ['tabulate',
+           'six',]
```
Comment (khalibartan) [inline pgmpy/extern/__init__.py]: @ankurankan Remove `,` before the closing braces.

File: pgmpy/extern/tabulate/__init__.py [hunk 2]
```diff
@@ -0,0 +1,3 @@
+from .tabulate import tabulate
+
+__all__ = ['tabulate',]
```
Comment (khalibartan) [inline pgmpy/extern/tabulate/__init__.py]: @ankurankan Remove `,` before the closing braces.

**Comment by khalibartan** [comment]
@ankurankan You can omit the `__init__` file but you will need to change how we are exporting them to other modules. IMO there is no need for it. Even I'm not able to understand reason :P.

**Comment by ankurankan** [comment]
Rather moved licenses inside the files. So closing this.

**Pull request closed without merging**